docs/World/constants.md

```diff
@@ -0,0 +1,3 @@
+# Constants
+
+This file will return you the values of Air Molar Mass, Gravitacional Acceleration and Air Gas Constant.
```

Comment:
I think a better explanation would be to say that there are several constants available there, and list the names of the constants and their values with a table, like so:

| Name | Value |
|------|-------|
| c1   | v1    |
| c2   | v2    |

You can generate such tables [with this website](https://www.tablesgenerator.com/markdown_tables).